src/components/ui/bottomNavigation.tsx

```diff
@@ -6,32 +6,60 @@ import { Home, Sprout, UserRound } from "lucide-react";
 import { Link } from "react-router-dom";
 
 export function BottomNavigationBar() {
-	const [value, setValue] = useState(0);
+  const [value, setValue] = useState(0);
 
-	return (
-		<Box sx={{ width: "100%", position: "sticky", bottom: 0 }}>
-			<BottomNavigation
-				showLabels
-				value={value}
-				onChange={(event, newValue) => {
-					setValue(newValue);
-					console.log(event);
-				}}
-			>
-				<BottomNavigationAction label="Home" icon={<Home />} component={Link} to="/" />
-				<BottomNavigationAction
-					label="Plants"
-					icon={<Sprout />}
-					component={Link}
-					to="/myPlants"
-				/>
-				<BottomNavigationAction
-					label="Profile"
-					icon={<UserRound />}
-					component={Link}
-					to="/profile"
-				/>
-			</BottomNavigation>
-		</Box>
-	);
+  return (
+    <Box sx={{ width: "100%", position: "sticky", bottom: 0 }}>
+      <BottomNavigation
+        value={value}
+        onChange={(event, newValue) => {
+          setValue(newValue);
+          console.log(event);
```

Comment:
Dient deze console.log() voor monitoring of was dit enkel bedoeld voor tijdelijke testing-purposes?

Reply:
Dit is omdat andere het project niet gebuild graakt, vite klaagt over dat het event niet wordt gebruikt. Maar al ik het weg doe het event dan klaagt het componenten